src/wallet/wallet.cpp

```diff
@@ -3809,8 +3809,7 @@ bool CWallet::CreateTransaction(const std::vector<CRecipient>& vecSend, CWalletT
                         }
                     }
 
-                    if (IsDust(txout, ::dustRelayFee))
-                    {
+                    if (IsDust(txout, ::dustRelayFee)) {
```

Comment:
pls revert this to original state to avoid adding unrelated changes

Reply:
Done in ec4cc32